src/monero_ux_nano.c

```diff
@@ -113,11 +90,7 @@ UX_STEP_CB(ux_menu_info_1_step, nn, ui_menu_info_action(0),
 UX_FLOW(ux_flow_info, &ux_menu_info_1_step);
 
 unsigned int ui_menu_info_action(unsigned int value __attribute__((unused))) {
```

Comment:
Drop the function and directly call `ui_menu_main_display()` when needed (nano and stax code).